snaps/reference/rpc-api.md

```diff
@@ -636,7 +636,7 @@ interact with the specified snaps.
 
 A website can also call this method to invoke the specified JSON-RPC method of the specified snap.
 
-This method is synonymous to [`wallet_invokeSnap`](#wallet_invokesnap), and is only callable by websites.
+This method is synonymous to [`wallet_invokeSnap`](#wallet_invokesnap).
```

Comment:
Can we move this change into another PR? I want to approve this and hold off on the other part (I want to re-write the how-to).